lib/dataTableHelpers.tsx

```diff
@@ -151,7 +151,9 @@ export function generateColumnsForDataSchema<T>(
 
     if (schemaDataById) {
         const dataSchema = schemaDataById[schemaDataId];
-        const dependencies = dataSchema ? dataSchema.requiredDependencies : [];
+        const dependencies = dataSchema
+            ? _.uniq(dataSchema.requiredDependencies)
```

Comment:
Seems like we have some duplicate dependencies in https://raw.githubusercontent.com/ncihtan/schematic/main/data/schema_org_schemas/HTAN.jsonld

Reply:
interesting - can you name some? I guess `Tumor Grade`?